gatling-jms/src/main/scala/io/gatling/jms/jndi/JmsJndiConnectionFactoryBuilder.scala

```diff
@@ -37,20 +39,24 @@ case class JmsJndiConnectionFactoryBuilderUrlStep(connectionFactoryName: String)
 case class JmsJndiConnectionFactoryBuilderFactoryStep(
     connectionFactoryName: String,
     url:                   String,
-    credentials:           Option[Credentials] = None
+    credentials:           Option[Credentials] = None,
+    properties:            Map[String, String] = Map.empty
 ) {
 
   def credentials(user: String, password: String) = copy(credentials = Some(Credentials(user, password)))
 
+  def property(key: String, value: String) = copy (properties = properties + (key -> value))
```

Comment:
I missed this one: `updated(key, value)` would be better IMO. I'll fix when I'll cherry-pick.